CHANGELOG.md

```diff
@@ -6,9 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.0]
+
+- [(GH-183)](https://github.com/uppetlabs/pdkgo/issues/183) `pct new` handles namespaced templates
```

Comment:
@sanfrancrisko do we need to be explicit about this not being backward compatible with 0.3.0 templates?